validator/validator-main.protoascii

```diff
@@ -357,6 +357,26 @@ tags: {
   }
   spec_url: "https://amp.dev/documentation/guides-and-tutorials/learn/spec/amphtml/#custom-fonts"
 }
+# Allowlisted CSS provider
+tags: {
+  html_format: AMP
+  enabled_by: "transformed"
+  tag_name: "LINK"
+  spec_name: "link rel=stylesheet for amp-story-1.0 css"
+  descriptive_name: "link rel=stylesheet for amp-story-1.0 css"
+  mandatory_parent: "HEAD"
+  attrs: {
+    name: "href"
+    mandatory: true
+    value_regex: "https://cdn\\.ampproject\\.org/v0/amp-story-1\\.0\\.css"
```

Comment:
@gmajoulet @jridgewell  should we be allowing rtv versions here?

Reply:
looks like the toolbox optimizer can rewrite urls to rtv/ for the script tags so the `link` should also be rewritten to the same rtv url for the css

Reply:
i retried testing transformed documents with a rtv url and it doesn't seem to be valid so i'm going to remove allowing rtv url's for the amp-story css link